libsolidity/codegen/ir/Common.h

```diff
@@ -49,6 +49,7 @@ struct YulArity
 
 struct IRNames
 {
+	static std::string externalFunctionPart(Declaration const& _functionOrVardecl);
```

Comment:
"part" sounds non-specific enough that I didn't really know what to expect from the PR name alone. Maybe we could have a more specific term to refer to it? How about something like "ABI wrapper"?

```suggestion
	static std::string externalFunctionABIWrapper(Declaration const& _functionOrVardecl);
```

Alternative ideas: `externalFunctionGlue`, `externalFunctionAdapter`.